File: Class/C language/For Loop Practice/pattern7.cpp
//Rhombus Pyramid
#include<Stdio.h>
main(){
	int num,i,j,k;
	
	printf("Enter number:");
	scanf("%d",&num);
	
	for(i=1;i<=num;i++){
		for(k=1;k<=i-1;k++){
			printf(" ");
		}
		for(j=1;j<=num;j++){
			printf("* ");
		}
		printf("\n");
	}
	
}

File: Python/class/Logical Program's/pattern7.py
# *****
# *   *
# *   *
# *   *
# *****
num=int(input("Enter a Number:-"))
for i in range(num):
    for j in range(num):
        if j==0 or j==num-1 or i==0 or i==num-1:
            print("* ",end="")
        else:
            print("  ",end="")
    print()